operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java

```diff
@@ -54,7 +54,13 @@
       + "exceeded configured domainPresenceFailureRetryMaxCount: "
       + DomainPresence.getDomainPresenceFailureRetryMaxCount()
       + " The domainPresenceFailureRetryMaxCount is an operator tuning parameter and can be controlled"
-      + " by adding it to the weblogic-operator-cm configmap.";
+      + " by adding it to the weblogic-operator-cm configmap."
+      + " To force the introspector to start retrying again, update 'domain.spec.introspectVersion'.";
+
+  String INTROSPECTION_ERROR = "Introspection Error: ";
+
+  String FATAL_ERROR_DOMAIN_STATUS_MESSAGE = "Introspection encountered a fatal error and will not retry automatically."
```

Comment:
Can we internationalize this by pulling from the resource bundle?

Reply:
Fixed in ea644db .